GameServerApp/Logic/GameServerConfig.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.IO;
+using Newtonsoft.Json.Linq;
+
+namespace LeagueSandbox.GameServerApp.Logic
+{
+    public class GameServerConfig
```

Comment:
This is in a better location now 👍 

I still don't like the name (referring to game server even though this isn't technically about the server) but I couldn't come up with much better ideas.

Reply:
Same.